lib/opal/nodes/top.rb

```diff
@@ -108,6 +180,34 @@ def compile_end_construct
       def version_comment
         "/* Generated by Opal #{Opal::VERSION} */"
       end
+
+      private
+
+      def generate_import_name(module_name)
+        # generate random import name for a ruby module_name. Also replaces some characters that are illegal in JS import names.
+        module_name.gsub('.', 'o_').gsub('-', '_').gsub('/', '_').gsub('@', '_at_') + rand(36**8).to_s(36)
+      end
+
+      def import_child_paths(import_lines, base_dir, module_path)
```

Comment:
Same for this method: it's a part of the compilation node, but it doesn't even use `self`. It should also be extracted @elia wdyt?